examples/utils_data.jl

```diff
@@ -563,6 +565,18 @@ function get_data_context2dturbulence(batchsize;
         xtest = real(xtest_upsampled)
     end
     
+    if filter isa Nothing
```

Comment:
We didnt end up using this in the color shift paper - just 512x512. do you want me to remove?